db/migrate/20200827204602_add_inbox_url_to_conversations.rb

```diff
@@ -0,0 +1,7 @@
+class AddInboxUrlToConversations < ActiveRecord::Migration[5.2]
+  def change
+    add_column :conversations, :parent_status_id, :bigint, null: true, default: nil
```

Comment:
No foreign key or index defined on the table. Not sure if we should do that to make it consistent with other columns in the database, or not because the conversations table is huge and the index would be huge too and the foreign keys make everything much slower and we should begin moving away from them.